cmd/containerd-nydus-grpc/main.go

```diff
@@ -44,40 +44,40 @@ func main() {
 			var snapshotterConfig config.SnapshotterConfig
 
 			if err := defaultSnapshotterConfig.FillUpWithDefaults(); err != nil {
-				return errors.New("failed to fill up nydus configuration with defaults")
+				return errors.New("failed to generate nydus default configuration")
 			}
 
 			// Once snapshotter's configuration file is provided, parse it and let command line parameters override it.
 			if snapshotterConfigPath != "" {
 				if c, err := config.LoadSnapshotterConfig(snapshotterConfigPath); err == nil {
 					// Command line parameters override the snapshotter's configurations for backwards compatibility
 					if err := config.ParseParameters(flags.Args, c); err != nil {
-						return errors.Wrap(err, "parse parameters")
+						return errors.Wrap(err, "failed to parse commandline options")
```

Comment:
In general, we tend to omit the words like `failed to` or `error` in a Golang wrapped errors chain in case there are too many `fail to` in final error message.

Reply:
It's at the top most level, so added `failed to`:)

Reply:
the returned error is not the topmost level, I am afraid. Its parent level is `err` returned from `ParseParameters()`

Reply:
True.